rs/ethereum/cketh/minter/src/state.rs

```diff
@@ -53,14 +55,12 @@ pub struct State {
     pub ethereum_network: EthereumNetwork,
     pub ecdsa_key_name: String,
     pub cketh_ledger_id: Principal,
-    pub eth_helper_contract_address: Option<Address>,
-    pub erc20_helper_contract_address: Option<Address>,
+    pub eth_log_scraping: LogScrapingState,
+    pub erc20_log_scraping: LogScrapingState,
```

Comment:
For a log scraping we need 2 info (contract address and last block scraped), which is now aggregated in `LogScrapingState`. I updated the tests in`rs/ethereum/cketh/minter/src/state/audit/tests.rs` to ensure we can replay events as expected and this state refactoring did not break anything